lms/djangoapps/grades/new/course_grade.py

```diff
@@ -65,6 +65,18 @@ def chapter_grades(self):
         }
 
     @lazy
+    def subsection_grades(self):
```

Comment:
Nit: Could we try to make this function and `locations_to_scores` have more similar names, since the data they return is so similar? Perhaps s/`locations_to_scores`/`problem_scores`?